test/test_tensorrec.py

```diff
@@ -115,6 +115,39 @@ def test_predict_item_repr(self):
         item_repr = self.unbiased_model.predict_item_representation(self.item_features)
         self.assertEqual(item_repr.shape, (self.item_features.shape[0], 10))
 
+    def test_predict_user_bias_unbiased_model(self):
+        self.assertRaises(
+            NotImplementedError,
+            self.unbiased_model.predict_user_bias,
+            self.user_features)
+
+    def test_predict_item_bias_unbiased_model(self):
+        self.assertRaises(
+            NotImplementedError,
+            self.unbiased_model.predict_item_bias,
+            self.item_features)
+
+
+class TensorRecMovielensPrediction(TestCase):
```

Comment:
This seems to be mis-named, now that you've swapped the dataset. Also, could you throw a #TODO in here to collapse this test class once the bug with biases is fixed?